src/EnergyPlus/RefrigeratedCase.cc

```diff
@@ -11038,6 +11038,9 @@ void RefrigCondenserData::UpdateCondenser(EnergyPlusData &state)
     // Set outlet node variables that are possibly changed
     state.dataLoopNodes->Node(this->OutletNode).Temp = this->OutletTemp;
 }
+void RefrigCondenserData::oneTimeInit([[maybe_unused]] EnergyPlusData &state)
+{
+}
```

Comment:
I get there will be some of these that are difficult to extract out the "one time" initialization, so this is fine.  I think as we carve up the component models it will become more clear.